man/systemd.special.xml

```diff
@@ -240,6 +240,9 @@
             names like <varname>single</varname>, <varname>rescue</varname>, <varname>1</varname>,
             <varname>3</varname>, <varname>5</varname>, …; see
             <citerefentry><refentrytitle>systemd</refentrytitle><manvolnum>1</manvolnum></citerefentry>.</para>
+
+            <para>Please note that using WantedBy=default.target for normal services is not a good idea,
+            since such a service will also be run on non-standard boots like on system update, selinux relabel…</para>
```

Comment:
LGTM in principle; maybe formulate it like a piece of actionable advise to the users, e.g. "Use WantedBy= to real targets instead of default.target, unless ..."

Reply:
I'd list emergency boot here, because it's a more universal concept, and maybe drop the selinux relabel item because it's very specific to some distros. (also it would be "SELinux" rather than "selinux" I think...

Reply:
> LGTM in principle; maybe formulate it like a piece of actionable advise to the users, e.g. "Use WantedBy= to real targets instead of default.target, unless ..."

I agree to some point, but I don't like "real" as a word here. Maybe say "explicit" or so (because default.target is less explicit, and more symbolic)